### PR TITLE
[Core] Adding GenerateEdges and GenerateFaces to Pyramid3D5N

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Adding GenerateEdges and GenerateFaces to Pyramid3D5N

**🆕 Changelog**

- [Adding GenerateEdges and GenerateFaces tp Pyramid3D5N](https://github.com/KratosMultiphysics/Kratos/commit/b1cc90f5e44186d1a5e2a9245fc9ad2146b548dd)
- Pragma once

### CODE REVIEW

**Comment by loumalouomega** [comment]
Ping @philbucher

File: kratos/geometries/pyramid_3d_5.h [hunk 1]
```diff
@@ -63,6 +64,13 @@ class Pyramid3D5 : public Geometry<TPointType>
     /// Geometry as base class.
     typedef Geometry<TPointType> BaseType;
 
+    /**
+     * Type of edge and face geometries
+     */
+    typedef Line3D2<TPointType> EdgeType;
+    typedef Triangle3D3<TPointType> FaceType1;
+    typedef Quadrilateral3D4<TPointType> FaceType2;
```
Comment (philbucher) [inline kratos/geometries/pyramid_3d_5.h:72]: Modern cpp is to use using over typedef

**Review by philbucher — APPROVED** [review]
Would be good if you added a test ❤️

**Comment by loumalouomega** [comment]
> Would be good if you added a test ❤️

Next pr

**Pull request merged**